NEW_MIPS_PROCESS.md

```diff
@@ -0,0 +1,329 @@
+            },
+            "source": {
+              "publicKey": "B62qre3erTHfzQckNuibViWQGyyKwZseztqrjPZBv6SQF384Rg6ESAy"
+            }
+          },
+          ...
+          {
+            "blockHeight": 255481,
+            "memo": "E4YM2vTHhWEg66xpj52JErHUBU4pZ1yageL4TVDDpTTSsv8mK6YaH",
+            "nonce": 105533,
+            "receiver": {
+              "publicKey": "B62qkiF5CTjeiuV1HSx4SpEytjiCptApsvmjiHHqkb1xpAgVuZTtR14"
+            },
+            "source": {
+              "publicKey": "B62qoXQhp63oNsLSN9Dy7wcF3PzLmdBnnin2rTnNWLbpgF7diABciU6"
+            }
+          }
+        ]
+      }
+    }
+    ```
+    Notice the base58 encoded memo field
+
+  4. Concatenate transactions for all canonical blocks in the voting period
+3. Filter the votes
+   - _memo.lower()_ exactly equal to _mip3_ or _no mip3_
+   - _source_ = _receiver_ (self transaction)
+4. The memo field is base58 encoded
+5. If there are multiple votes associated with a single public key, only the _latest_ vote is counted;
+  _latest_ being defined:
+   - For multiple votes from the same account across several blocks, take the vote in the highest block.
+   - For multiple votes from the same account in the same block, take the vote with the highest nonce.
+
+### Calculate weighted voting results
+  a. Sum all aggregated voter stake to get the total voting stake
+  b .For each delegate, start with their total stake, and subtract the balances of accounts that delegate
+  to them with an overriding vote
+  c. Divide yes/no vote stakes by the total voting stake
+
+### Adjust Votes and Voting Stake with Non-Delegating Voters
+
+Find all votes made by a delegating account, and subtract their account balance from the final voting stake if they disagree with their delegate
+
+```
+delegating_stake = {}
+delegating_votes = {}
+for vote in votes:
+    if vote.pk in delegators:
+        delegating_stake[vote.pk] = accounts[vote.pk]['balance']
+        delegating_votes[vote.pk] = vote.memo
+for vote in delegating_votes
+    delegate_vote = votes[accounts[pk]['delegate']]
+    if against(delegate_vote) and for(vote) and pk not in delegating_votes:
+        no_stake -= delegating_stake[vote.pk]
+    else if for(delegate_vote) and  against(vote) and pk not in delegating_votes:
+        yes_stake -= delegating_stake[vote.pk]
+```
+
+Check agreement with the voting results dashboard and/or @trevorbernard’s verification scripts
+
+#### Vote Verification Scripts
+  - [MIP3](https://gist.github.com/trevorbernard/ec11db89bb9079dd0a01332ef32c0284)
+  - [MIP4](https://gist.github.com/trevorbernard/928be21e8e1d9464c3a9b2453d9fd886)
+
+MIP3 and MIP4 Voting Results
+- [MIP3 results dashboard](https://ocv.minaprotocol.com/proposal/1/results)
+- [MIP4 results dashboard](https://ocv.minaprotocol.com/proposal/2/results)
+
+Conclusion
+MIP voting is a substantial addition to Mina’s ever-evolving suite of on-chain governance
```

Comment:
Remove this conclusion, and add credit top graonola team for the original implementation